.claude/agents/kfc/spec-test.md

```
@@ -0,0 +1,107 @@
+
+**测试步骤**:
+1. [步骤1]
+2. [步骤2]
+3. [验证点]
+
+**预期结果**:
+- [预期结果1]
+- [预期结果2]
+
+[更多测试用例...]
+
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)